mypyc/irbuild/mapper.py

```diff
@@ -123,6 +124,8 @@ def type_to_rtype(self, typ: Type | None) -> RType:
                 return tuple_rprimitive
         elif isinstance(typ, CallableType):
             return object_rprimitive
+        elif isinstance(typ, ParamSpecType):
+            return object_rprimitive
```

Comment:
If it's `P.args` or `P.kwargs`, we use `tuple[object, ...]` and `dict[str, object]` as the upper bound of the ParamSpecType (respectively). Does that imply we should do something similar to the TypeVarType branch below?

```suggestion
            return self.type_to_rtype(typ.upper_bound)
```

Reply:
Good point, thanks!